### PR TITLE
js-oop

### PR DESCRIPTION
**Pull request by vadymkotlyarenko**
_Complete the form below and remove this line._

### js-oop

[Demo](https://vadymkotlyarenko.github.io/frontend-nanodegree-arcade-game/) |
[Code base](https://github.com/vadymkotlyarenko/frontend-nanodegree-arcade-game/blob/master/js/app.js)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/vadymkotlyarenko/task_js_oop/app.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+var Enemy = function(x, y, speed) {
```
Comment (AMashoshyna) [inline submissions/vadymkotlyarenko/task_js_oop/app.js]: Please use ES6 features (`let`/`const` behave more predictable than `var`s)

File: submissions/vadymkotlyarenko/task_js_oop/app.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    this.sprite = 'images/enemy-bug.png';
+    this.x = x;
+    this.y = y;
+    this.speed = speed;
+};
+
+Enemy.prototype.update = function(dt) {
+    this.x += dt * this.speed;    
+    if(Math.abs(this.x - player.x) < 70 && Math.abs(this.y - player.y) < 50) {
+        player.x = 200;
```
Comment (AMashoshyna) [inline submissions/vadymkotlyarenko/task_js_oop/app.js]: Numbers without explanation in code are often called "magic numbers". They make code harder to understand for other developers or event yourself after a period of time.
Please put the numbers into properly named constants.

**Comment by AMashoshyna** [comment]
Please rename your PR according to task name.

**Comment by OleksiyRudenko** [comment]
@vadymkotlyarenko Please, change the PR title so it gives a better idea of what's inside right from all PRs listing. Thanks.

**Comment by AMashoshyna** [comment]
Well done!

**Pull request merged**